src/strongbox/github_api.clj

```diff
@@ -120,51 +129,58 @@
+                  (and single-asset? single-game-track?) {:game-track (first known-game-tracks)
+                                                          :version (track-version version (first known-game-tracks))  :-mo :sa--1gt}
 
-                  ;; multiple assets, no game track present in file name, single known game track. use that.
-                  ;; ambiguous case, other assets may have game track in their file name
-                  ;; this or other assets may be variations of the 'main' addon, like '-nolib' ?
+                  ;; single asset, no game track present in asset name or release name with multiple entries in `:game-track-list`.
+                  ;; assume all entries in `:game-track-list` supported.
+                  (and single-asset? many-game-tracks?) (vec (for [game-track known-game-tracks]
+                                                               {:game-track game-track, :version (track-version version game-track) :-mo :sa--Ngt}))
+
+                  ;; multiple assets, no game track present in asset name or release name, and no `:game-track-list`.
+                  ;; default to `:retail` and make a note that we're guessing in the face of ambiguity.
+                  ;; other assets may have a game track in their asset name.
+                  (and many-assets? no-known-game-tracks?) (do (debug (format "no game track detected for release '%s' and asset '%s', assuming 'retail'"
+                                                                              (:name release) (:name asset)))
+                                                               {:game-track :retail :version version :-mo :ma--ngt})
+
+                  ;; multiple assets, no game track present in asset name or release name with a single entry in `:game-track-list`. use that.
+                  ;; other assets may have a game track in their asset name.
+                  ;; this or other assets may be variations of the 'main' addon, like `-nolib`.
                   (and many-assets? single-game-track?) {:game-track (first known-game-tracks)
                                                          :version (track-version version (first known-game-tracks)) :-mo :ma--1gt}
 
-                  ;; multiple assets, no game track present in file name, multiple known game tracks. default to :retail
-                  ;; ambiguous case, other assets may have game track in their file name.
-                  (and many-assets? many-game-tracks?) {:game-track :retail :version version :-mo :ma--Ngt}
+                  ;; multiple assets, no game track present in asset name or release name with multiple entries in `game-track-list`.
+                  ;; treat the same as single-asset+many-game-tracks and assume all entries in `:game-track-list` are supported.
+                  ;; other assets may have game track in their file name.
+                  (and many-assets? many-game-tracks?) (vec (for [game-track known-game-tracks]
```

Comment:
um, perhaps we should consider getting rid of these many-assets/single-asset conditions? especially if there is nothing distinct about how we treat an asset if there are  other assets in a release (besides the ad-hoc :-mo classification).